tox.ini

```diff
@@ -25,7 +25,9 @@ skipsdist=True
 [testenv]
 setenv=
     SIDEBOARD_CONFIG_OVERRIDES=test-defaults.ini
-deps= -rrequirements.txt
+deps=
+    -rrequirements.txt
+    -rtest_requirements.txt
```

Comment:
Adding this line seemed to do the trick.

I like how things are being handled in the ``ubersystem`` plugin (https://github.com/magfest/ubersystem/blob/master/requirements_test.txt#L1) where ``requirements_test.txt`` directly references ``requirements.txt`` but that would require some additional changes to how we write our ``setup.py`` file which would further affect our local Jenkins jobs and such, so I'd prefer to not take that on as part of this merge.